pkg/election/election.go

```diff
@@ -91,11 +91,21 @@ type Election struct {
 
 	closed sync2.AtomicInt32
 	cancel context.CancelFunc
-	bgWg   sync.WaitGroup
+
+	campaignWg sync.WaitGroup
```

Comment:
Why do we rename it to `campaignWg`? BTW, another `compaignWg` defined in L220.

Reply:
ok, I change it back to `bgwg`